src/assets/scss/application.scss

```diff
@@ -239,6 +239,36 @@ $nav-active-border-thickness: 5px;
   }
 }
 
+.account-header-navigation__button, .account-navigation__button {
+  border: 0 none;
+  cursor: pointer;
+  background-color: unset;
+}
+
+.account-header-navigation__button {
+  @include govuk-font($size: 16, $weight: bold);
+  color: govuk-colour(white);
+  padding: 0;
+  text-decoration: none;
+  position: absolute;
+  top: govuk-spacing(4);
+  right: 0;
+  margin: 0;
+  &:hover {
+    text-decoration: solid underline 3px;
```

Comment:
I couldn't use mixins for this particular bit because it says:
   // Avoid using the `govuk-link-common` mixin because the links in the header
    // get a special treatment, because:
    //
    // - underlines are only visible on hover
    // - all links get a 3px underline regardless of text size, as there are
    //   multiple grouped elements close to one another and having slightly
    //   different underline widths looks unbalanced

So I used standard govuk-frontend classes instead